docs/sql-performance-tuning.md

```diff
@@ -114,6 +114,28 @@ that these options will be deprecated in future release as more optimizations ar
     </td>
     <td>1.1.0</td>
   </tr>
+  <tr>
+    <td><code>spark.sql.sources.parallelPartitionDiscovery.threshold</code></td>
+    <td>32</td>
+    <td>
+      Configures the threshold to enable parallel listing for job input paths. If the number of
+      input paths is larger than this threshold, Spark will use parallel listing on the driver side.
```

Comment:
> Spark will use parallel listing on the driver side.

It sounds like Spark just runs parallel listing on the driver side using multi-threading. How about using "Spark will list the files by using Spark distributed job"?

Reply:
Will do.